requirements.txt

```diff
@@ -0,0 +1,5 @@
+pytest
```

Comment:
Please remove pytest and pycodestyle as we only need them in the travis.yml

Reply:
about pytest and pycodestyle: we'll have to keep the pycodestyle in the file else we'll have error about "pycodestyle not installed" or something like that ?

Reply:
Done